js-exercises/update-object/updateObject.js

```diff
@@ -0,0 +1,29 @@
+
+function updateObject(...args) {
```

Comment:
Why not put your params here itself like this? 
```suggestion
function updateObject(position, replaceValue, arr) {
```